lib/validator/validator.go

```diff
@@ -30,14 +32,35 @@ import (
 var validate *validator.Validate
 
 var (
-	nameRegex          = regexp.MustCompile("^[a-zA-Z0-9_.-]+$")
+	nameRegex          = regexp.MustCompile("^[a-zA-Z0-9_.-]{1,128}$")
 	interfaceRegex     = regexp.MustCompile("^[a-zA-Z0-9_-]{1,15}$")
-	labelRegex         = regexp.MustCompile("^[a-zA-Z_./-][a-zA-Z0-9_./-]*$")
+	labelRegex         = regexp.MustCompile("^[a-zA-Z_./-][a-zA-Z0-9_./-]{0,127}$")
```

Comment:
Nope the label regex has an initial char [a-zA-Z_./-] followed by 0-127 [a-zA-Z0-9_./-]

Basically, the first char cannot be a number - the 0,127 applies to the second character